### ISSUE TITLE
Adding neural voices to OptiKey

### ISSUE THREAD

**Issue by williamluer** [OptiKey/OptiKey#870]
I would like to add custom voice clones from a neural TTS model to OptiKey. This way people would be able to get a custom voice clone created and then communicate in their own voice.

Has anybody added new voices to OptiKey before so that they can be used like the other voices? Are there any resources or documentation I can look at? Thank you!

**Comment by kmcnaught** [comment]
Hi @williamluer, Optikey currently supports SAPI5 voices, including those you can download from Model Talker - see https://github.com/OptiKey/OptiKey/wiki/Free-TTS-Voices. 

@zelmon64 has also added support for MaryTTS which uses an HTTP interface, this might be similar to what you would need for a neural TTS model? You can see old discussion here: https://github.com/OptiKey/OptiKey/issues/334 and the implementation here: https://github.com/OptiKey/OptiKey/blob/437435e89baa7b709a7cb5805fe0a66120a812e2/src/JuliusSweetland.OptiKey.Core/Services/AudioService.cs

**Comment by williamluer** [comment]
Hi @kmcnaught thank you for passing along that information! The HTTP interface created for MaryTTS seems like it could definitely be a possibility for neural voices. I think it would be preferred if we could encode the neural voice as a SAPI5 voice. Have you ever created a new SAPI5 voice before?

**Comment by kmcnaught** [comment]
@williamluer no, I'm afraid I haven't ever created SAPI5 voices and don't know about how you would do that.

**Comment by bigmutty** [comment]
This wouldn't be a pretty solution, but what if we created a fullscreen keyboard, have a blank Microsoft Word document opened and active, use the read aloud feature in Word, and have a key that will clear the document.
